tests/product/base_product_case.py

```diff
@@ -124,16 +122,25 @@ def setup_docker_cluster(self, cluster_type='centos'):
 
     def tearDown(self):
         self.restore_stdout_stderr_keep_open()
```

Comment:
We should also delete the dist directory at the end of the test since they aren't reused, so we don't leave all these extra directories lying around.